webapp/lib/charts.dart

```diff
@@ -204,4 +204,27 @@ class DailyTimeseriesLineChartView {
     var chartConfig = new chartjs.ChartConfiguration(type: 'line', data: chartData, options: chartOptions);
     chart = chartjs.Chart(canvas.getContext('2d'), chartConfig);
   }
- }
+}
+
+class SystemMetricsTimeseriesLineChartView extends DailyTimeseriesLineChartView {
+  @override void updateChart(List<Map<DateTime, num>> updatedCountsAtTimestampList, {String timeScaleUnit = 'day', num upperLimit}) {
```

Comment:
I think the style guide recommends having the `@override` on the line above, as such:
```suggestion
  @override
  void updateChart(List<Map<DateTime, num>> updatedCountsAtTimestampList, {String timeScaleUnit = 'day', num upperLimit}) {
```

Reply:
Good to know. Changed it accordingly.